src/ophyd_async/panda/_common_blocks.py

```diff
@@ -22,6 +23,39 @@ class PulseBlock(Device):
     width: SignalRW[float]
 
 
+class PcompDirectionOptions(str, Enum):
+    positive = "Positive"
+    negative = "Negative"
+    either = "Either"
+
+
+class PcompRelativeOptions(str, Enum):
+    absolute = "Absolute"
+    relative = "Relative"
+
+
+EnableDisableOptions = SubsetEnum["ZERO", "ONE"]
+
+
+class PcompBlock(Device):
+    active: SignalR[bool]
+    dir: SignalRW[PcompDirectionOptions]
+    enable: SignalRW[EnableDisableOptions]
+    enable_delay: SignalRW[int]
+    health: SignalR[str]
+    inp: SignalRW[str]
+    label: SignalRW[str]
+    out: SignalRW[bool]
+    pre_start: SignalRW[int]
+    produced: SignalR[int]
+    pulses: SignalRW[int]
+    relative: SignalRW[PcompRelativeOptions]
+    start: SignalRW[int]
+    state: SignalR[str]
+    step: SignalRW[int]
+    width: SignalRW[int]
```

Comment:
We only want to put the things that we set from code here, so start, step, width, not enable_delay, pre_start, etc.